backend/alembic/versions/2024_11_10_1814-03585c5cb5f3_updated_listings_model.py

```diff
@@ -21,10 +21,27 @@
 
 def upgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
-    op.add_column("listings", sa.Column("tagline", sa.String(), nullable=False))
-    op.add_column("listings", sa.Column("location", sa.String(), nullable=False))
-    op.add_column("listings", sa.Column("image", sa.String(), nullable=False))
-    op.add_column("listings", sa.Column("price", sa.Float(), nullable=False))
+    op.add_column(
+        "listings",
+        sa.Column(
+            "tagline", sa.String(), server_default="default_tagline", nullable=False
+        ),
+    )
+    op.add_column(
+        "listings",
+        sa.Column(
+            "location", sa.String(), server_default="default_location", nullable=False
+        ),
+    )
+    op.add_column(
+        "listings",
+        sa.Column(
+            "image", sa.String(), server_default="default_image_url", nullable=False
+        ),
+    )
+    op.add_column(
+        "listings", sa.Column("price", sa.Float(), server_default=0.0, nullable=False)
+    )
```

Comment:
I'm reasonably sure this should work.

Based on only changing it in alembic you'd handle the db migration, but still have a required input at the app layer. I think.